site.yml

```diff
@@ -76,6 +81,19 @@
       copy: src=jobs dest=/var/lib/jenkins/jobs/default-dsl-job/workspace directory_mode=0750 mode=0640 owner=jenkins group=jenkins
     - name: generate jenkins job from dsl seed job
       shell: "curl -X POST 'http://127.0.0.1:8080/job/default-dsl-job/build' -k $(cat ~/jenkins_auth)"
+    - name: create jenkins .ssh directory
+      file: path=/var/lib/jenkins/.ssh state=directory mode=0775 owner="jenkins"
+    - name: deploy deployer keys
+      copy: content="{{ deployer_key['key'] }}" dest=/var/lib/jenkins/.ssh/{{ deployer_key['name'] }} mode=0600 owner="jenkins"
```

Comment:
How much work is it to re-test this particular part? Could you tell by applying it in Vagrant and looking at the contents of the file?

I presume the different rendering would be caused by using a map/hash/dict (whatever they're called here)? What about using the folded scalars approach described in this article (which also has a good account of why it's a good thing to wrap):

https://servercheck.in/blog/yaml-best-practices-ansible-playbooks-tasks